pgx-examples/aggregate/src/lib.rs

```diff
@@ -0,0 +1,122 @@
+use pgx::*;
+use std::{
+    str::FromStr,
+    ffi::CStr,
+};
+
+pg_module_magic!();
+
+
+use serde::{Deserialize, Serialize};
```

Comment:
Is this where we want this import?

Reply:
I keep forgetting we don't run `rustfmt` on CI...